### PR TITLE
Support Atom as an output format

### PR DESCRIPTION
**Pull request by peterjanes**

### CODE REVIEW

**Comment by TobiX** [comment]
First, thanks for your contribution. As it stands, I won't merge it, as it doesn't even build on Travis. It would be nice if you could remove the pytz dependency, since packaging modules using pkg_resources with PyInstaller (for Windows-One-EXE-builds) is a pain (OTOH, it seems PyInstaller has a hook for pytz, so it might not be that bad after all).

And finally: What is the rationale behind this change? What does Atom give me that RSS can't?

**Comment by peterjanes** [comment]
Last first: my aggregator escaped the contents of the RSS feed so all I saw was HTML source. The Atom version worked with no problem.

Not sure why it wouldn't build; is it just the missing dependency?  I don't think anything I did would break tests.

**Comment by blade2005** [comment]
```
E   ImportError: No module named pytz
```
From travis. I think you need to add into the travis build installing pytz. That being saidI would second @TobiX and say that if you don't absolutely need it then remove it.

**Comment by peterjanes** [comment]
I completely forgot that I removed `pytz` in my master branch but didn't merge it to the `atom` branch. This one should be better. (And I think I've managed to figure out how to get the dependencies into Travis.)

**Comment by TobiX** [comment]
Thanks for the contribution. I still have some issues with this feature:

- Please make the dependency on python-dateutil optional (see setup.cfg [extras] and tox.ini) - It's okay to fail gracefully if someone tries to use the atom output format and the module is not present.
- Please rebase your changes on top of the current master and remove all unrelated changes

File: dosagelib/comic.py [hunk 1]
```diff
@@ -106,8 +106,8 @@ def save(self, basepath):
         fn = fnbase + self.ext
         # compare with >= since content length could be the compressed size
         if os.path.isfile(fn) and os.path.getsize(fn) >= self.contentLength:
-            self._exist_err(fn)
-            return fn, False
+            out.info(u'Skipping existing file "%s".' % fn)
+            getHandler().comicDownloaded(self, fn)
```
Comment (TobiX) [inline dosagelib/comic.py]: This looks buggy to me. Does this work with all the different output handlers? Where is the previous code?

**Pull request closed without merging**